frontend/src/app/projects/project-detail/project-details.component.html

```diff
@@ -5,17 +5,20 @@
 
 <div class="flexbox">
   <div class="item">
-    <app-project-metadata
-      *ngIf="projectService.project"
-      [project]="projectService.project"
-    ></app-project-metadata>
-    <mat-divider></mat-divider>
+    <div
+      *ngIf="projectService.project && projectUserService.verifyRole('manager')"
+    >
+      <app-project-metadata
+        [project]="projectService.project"
+      ></app-project-metadata>
+      <mat-divider></mat-divider>
+    </div>
```

Comment:
Imo, the project metadata should be visible by any user, but only editable by managers.

Reply:
Or at least the description in a card.